docs/CHANGELOG.asciidoc

```diff
@@ -24,6 +24,8 @@
 
 === Bug Fixes
 
+Fix NullPointerException when creating or deleting a snapshot ({pull}30140[#30140])
```

Comment:
can you extend this a little? e.g. `Fix NullPointerException when creating or deleting a snapshot on a repository that has been written to by an older Elasticsearch after writing to it with a newer Elasticsearch version.`

Reply:
Thanks!